### ISSUE TITLE
Time to remove doc on rtd for each project

### ISSUE THREAD

**Issue by wkentaro** [jsk-ros-pkg/jsk_docs#4]
Already there are some documents

- jsk_recognition
- jsk_visualization
- jsk_common

@k-okada 
when you think it's time to remove these, please notify me.

**Comment by k-okada** [comment]
no, no, no, some of the package that have active maintainer can continue
using rtd in doc directory, What I did is try to include package with lazy
maintainers.

--
◉ Kei Okada

**Comment by wkentaro** [comment]
I mean removing site on rtd not doc directory, and adding link to each projects' top README which appoints jsk_docs' rtd.
Is this clear?
